DataHub/UserInterface/02a_Attributes.md

```diff
@@ -0,0 +1,259 @@
+
+- [DELETE]   
+  Click this button to delete the selected attribute. This button is only displayed if the checkbox of an attribute is selected.       
+  For detailed information, see [Delete an attribute](../Integration/01_ManageAttributes.md#delete-an-attribute).
+
+The list displays all attributes. Depending on the settings, the displayed columns may vary. All fields are read-only.
+
+- *Name*   
+  Attribute name.
+
+
+- *Status*   
+  Attribute status. The following statuses are available:
+  - ![Status](../../Assets/Icons/Status01.png "[Status]") **Active**
+  - ![Status](../../Assets/Icons/Status04.png "[Status]") **Inactive**   
+
+
+- *Group*   
+  Attribute group assigned to the attribute. If the attribute is not assigned to any attribute group, *not assigned* is displayed.
+
+- *Key*   
+  Attribute key.
+
+- *Data type*   
+  Attribute data type.   
+  For detailed information about all data types, see [Data type list](./04_DataTypeList.md).
+
+- *Description*   
+  Description to the attribute.
+
+
+- *Multi-language*   
+  Indication whether the attribute is multi-language or not:
+  - ![Check](../../Assets/Icons/Check.png "[Check]") (Checkmark): The attribute is multi-language.  
+  - ![Cross](../../Assets/Icons/Cross02.png "[Cross]") (Cross): The attribute is single language.
+
+
+- *Multi-channel*   
+  Indication whether the attribute is multi-channel or not:
+  - ![Check](../../Assets/Icons/Check.png "[Check]") (Checkmark): The attribute is multi-channel.  
+  - ![Cross](../../Assets/Icons/Cross02.png "[Cross]") (Cross): The attribute is single channel.
+
+
+- *Plugin*   
+  Module or plugin that owns the attribute.
+
+- *ID*   
+  Attribute identification number. The ID number is automatically assigned by the system.
+
+- *Modified on*   
+  Date and time of the last modification.
+
+- *Modified by*   
+  Name and username of the user who modified the attribute.
+
+- *Created on*   
+  Date and time of the creation.
+
+- *Created by*   
+  Name and username of the user who created the attribute.
+
+- *Name (Language)*   
+  Attribute name in the selected language. A single column is displayed for the system languages *English (United States)* and *Deutsch (Deutschland)*.
+
+- *Description (Language)*   
+  Attribute description in the selected language. A single column is displayed for the system languages *English (United States)* and *Deutsch (Deutschland)*.
+
+- ![Add](../../Assets/Icons/Plus01.png "[Add]") (Add)   
+  Click this button to create an attribute. The *Create attribute* view is displayed.   
+
+
+
+## Create attribute
+
+*DataHub > Settings > Tab ATTRIBUTES > Button Add*
+
+![Create attribute](../../Assets/Screenshots/DataHub/Settings/Attributes/CreateAttribute.png "[Create attribute]")
+
+**Create attribute**
+
+- ![Back](../../Assets/Icons/Back02.png "[Back]") (Back)   
+  Click this button to close the *Create attribute* view and return to the attributes list. All changes are rejected.
```

Comment:
```suggestion
  Click this button to close the *Create attribute* view and return to the attribute list. All changes are rejected.
```
Vgl. Header oben Attribute list -> Das haben wir mit allen Listen wahrscheinlich (attributes / attribute list = list of attributes). Also, singular oder plural, wenn vor dem Namen?